src/modules/taskStatus.js

```diff
@@ -0,0 +1,17 @@
+import { TasksClass } from './tasksClass.js';
+
+//  get task status on checkbox
+export const retainCheck = () => {
+  //  get the elements
+  const taskEls = document.querySelectorAll('.task-select-input');
+  const checkStatus = new TasksClass();
+  // get index and checkbox value by iterating and set checkbox
+  taskEls.forEach((theEl, theElIndex) => {
+    // theEl.checked = true;
```

Comment:
As part of the [ruby best practices](https://github.com/microverseinc/curriculum-ruby/blob/main/articles/ruby_best_practices.md) kindly remove this commented code from your codebase. Leaving commented code in your codebase is generally not a good practice. Commented-out code can clutter your codebase, make it harder to read, and can potentially lead to confusion or outdated information. If you need to review a previous version of your code, you can always use git history. :pray:

Reply:
I appreciate the reminder.